cfme/fixtures/v2v.py

```diff
@@ -152,6 +152,9 @@ def _form_data(second_provider, provider):
 @pytest.fixture(scope='function')
 def form_data_multiple_vm_obj_single_datastore(request, appliance, second_provider, provider):
     # this fixture will take list of N VM templates via request and call get_vm for each
+    form_data_multiple_vm_obj_single_datastore = namedtuple(
```

Comment:
**Required** This tuple should be defined at the module level and instantiated for each method.

Reply:
I thought of that, but wanted to let it come from reviewers as I didn't know it will be accepted.